lib/odbc/src/odbc.erl

```diff
@@ -1374,3 +1375,15 @@ string_terminate_value(null) ->
 
 port_timeout() ->
   application:get_env(?MODULE, port_timeout, ?ODBC_PORT_TIMEOUT).
+
+unwrap_conn_str({Cmd0, ConnStrWrapped}) ->
+    ConnStr = unwrap_secret(ConnStrWrapped),
+    [Cmd0, ConnStr];
```

Comment:
Is it ok that we change the structure of the list? I.e. return `[Cmd0, ConnStr]` instead of `Cmd0 ++ [ConnStr]`